update/Dockerfile

```diff
@@ -50,6 +50,12 @@ RUN --mount=type=secret,id=STEAM_USERNAME \
         linux32/libvstdlib_s.so \
         linux32/steamclient.so \
         linux32/steamconsole.so \
-        update_hosts_cached.vdf \
-        /root/.steam \
-        /root/Steam
+        update_hosts_cached.vdf; \
+    cd /root && rm -rf \
+        .steam; \
+    if [ "$APPID" = 90 ]; then \
+        # Keep `/root/Steam/appcache` to prevent `steamcmd` from re-downloading `hlds` games from scratch whenever the game is updated
+        rm -rf $( find Steam -maxdepth 1 | grep -vE '^Steam$|^Steam/appcache' ); \
+    else \
+        rm -rf Steam; \
+    fi
```

Comment:
this is difficult to understand and maintain. Simpler to use `rm -rf` and specify the directories to remove.

Reply:
would probably be easier to understand with full paths used instead

Reply:
yes.